src/services/project.service.js

```diff
@@ -33,7 +39,15 @@ const upload = (formData, conf) => httpService
     'Content-Type': 'multipart/form-data',
   }, conf)
   .then(({ data }) => data)
-  .catch((err) => Promise.reject(err.response.data));
+  .catch((err) => {
+    if (err.response.data.errors) {
+      Swal.fire({
+        icon: 'error',
+        title: 'Error',
+        text: err.response.data.errors[0],
+      });
```

Comment:
Are you sure all the possible error format?
Sometimes err.response.data.errors[0] can throw another issues
We can have different types of errors: server issue, network issue, laravel standard error, custom error we made and etc

Reply:
this is ok for now. we also have client-side validation

Reply:
I don't think so
For example I think if you try to do it after disconnect network, try to call this action, it would throw some errors

Reply:
Good, don't we revert changes here?

Reply:
It just need to reject